e2e/serial/swap/2_swapFlow2.test.ts

```diff
@@ -44,15 +45,10 @@ let driver: WebDriver;
 
 const browser = process.env.BROWSER || 'chrome';
 const os = process.env.OS || 'mac';
-const isFirefox = browser === 'firefox';
 
```

Comment:
We might want to keep any of these conditionals in case we turn Firefox e2e back on at some point

Reply:
I think previously we mentioned turning FF back on only for essential flows in the future (onboarding, etc). So I don't think we necessarily need these unused conditionals in the swaps flow, its just dead code atm. Happy to add them back if needed ever.